sdks/python/apache_beam/yaml/yaml_provider.py

```diff
@@ -651,6 +651,16 @@ class WindowInto(beam.PTransform):
     See [the Beam documentation on windowing](https://beam.apache.org/documentation/programming-guide/#windowing)
     for more details.
 
+    Sizes, offsets, periods and gaps (where applicable) must be defined using
+    a time unit suffix 's', 'm', 'h' or 'd' for seconds, minutes, hours
+    or days, respectively.
```

Comment:
Mention that if a suffix is omitted, we default to seconds.

Reply:
done.